### DISCUSSION TITLE
lammps cpu mpirun test

### DISCUSSION THREAD

**Discussion by chen-jialan** [mir-group/pair_nequip_allegro#15] [Q&A]
I'm testing the example, and successful. 
However, when I build the slab model for test, and mpirun is error (Caught signal 11 (Segmentation fault: invalid permissions for mapped object at address 0x3448b30)). Therefore, if or not test the slab model?

command: mpirun -n 24 $lammps -in md.in

compiler: oneapi

**Comment by Linux-cpp-lisp** [comment]
I believe we do not yet support systems where there are empty domains, such as slabs with a lot of vacuum, and that this can manifest in a segfault... is this correct @anjohan ?

**Reply by anjohan** [reply]
I've never tested, but if one of the processors has an empty domain, there may be an issue (use the `balance` or `processors` command to resolve - this is either way something you want to avoid).
